recipes/pygtftk/meta.yaml

```
@@ -1,16 +1,16 @@
-{% set version = "1.2.7" %}
+{% set version = "1.3.0" %}
 
 package:
   name: pygtftk
   version: '{{ version }}'
 
 source:
-  url: 'https://github.com/dputhier/pygtftk/archive/{{ version }}.tar.gz'
-  sha256: f3d58776a6ff3d79c43bf207a2157861ca2b5668f7f507dcca88994df6d77b4d
+  url: 'https://github.com/dputhier/pygtftk/archive/v{{ version }}.tar.gz'
+  sha256: ca28ae5a394e1be422eeafbcd686e9cb616197c6758aa16049ac201d254ff61d
 
 build:
   number: 2
```

This should satisfy the linter:
```suggestion
  number: 0
```